### PR TITLE
update README && upgrade Dockerfile golang:1.9.2

### PR DESCRIPTION
**Pull request by yangwenmai**

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @yangwenmai

File: README.md [hunk 1]
```diff
@@ -88,16 +88,13 @@ docker pull pingcap/pd
...
 ```bash
-# Set correct HostIP here. 
-export HostIP="192.168.199.105"
-
 docker run -d -p 2379:2379 -p 2380:2380 --name pd pingcap/pd \
           --name="pd" \
           --data-dir="pd" \
           --client-urls="http://0.0.0.0:2379" \
-          --advertise-client-urls="http://${HostIP}:2379" \
+          --advertise-client-urls="http://0.0.0.0:2379" \
```
Comment (siddontang) [inline README.md]: IMO, it is not good to assign 0.0.0.0 to the advertise url.

Comment (yangwenmai) [reply README.md]: Run a single node with Docker.

It should be 

```bash
docker run -d -p 2379:2379 -p 2380:2380 --name pd pingcap/pd \
          --name="pd" \
          --data-dir="pd" \
          --client-urls="http://0.0.0.0:2379" \
          --peer-urls="http://0.0.0.0:2380"
```

Comment (siddontang) [reply README.md]: @yangwenmai 

You can mount the PD port to your host port and set the host:mounted_port as the advertised URL.

Using 0.0.0.0 as the advertised URL is not recommended.

Comment (yangwenmai) [reply README.md]: 😓
The reason I am wrong here is not that I did not mount the PD port to my host port, but my docker network and host communication problems.

**Comment by siddontang** [comment]
LGTM 

PTAL @disksing 

Thank you very much @yangwenmai

**Pull request merged**